assessments/0002-notary-project.md

```diff
@@ -0,0 +1,195 @@
+- [ ] Setup a [Hugo website](https://gohugo.io) with a [Docsy theme](https://www.docsy.dev/)
+- [ ] Setup Netlify for hosting/CI
+- [ ] Develop a [maintainers list](https://github.com/notaryproject/notaryproject.dev/blob/main/.github/settings.yml) for the documentation and its repository (this has been started https://github.com/notaryproject/notaryproject/issues/77)
+- [ ] Develop a maintenance plan for the documentation and its repository 
+- [ ] Follow the [CNCF website guidelines checklist](https://github.com/cncf/techdocs/blob/main/howto/website-guidelines-checklist.md) for other required elements
+
+_Note_: [Notary Project branding artwork](https://github.com/cncf/artwork/tree/master/projects/notary) exists. Other branding elements, like colour schemes and whatnot, will need to be developed.
+
+### Information Architecture
+
+The content to be written could have the following structure (as a start - modifications will need to be made as development continues):
+
+**Site structure**
+
+- Landing page
+- Community
+   - Governance
+   - Community (details)
+   - Contributing
+      - Where to start
+      - How we work
+      - Filing and working on issues for the project
+      - Where the community is (contacts, meetings)
+      - Contributing to docs
+      - Where to ask for more help contributing
+      - Next steps
+- About
+- Blog (Get the word out about the project and how it's progressing)
+- Docs
+   - This section to be planned with the Notary Maintainers team
+   - Getting Started
+      - This section to be planned with the Notary Maintainers team
+
+**Topnav**
+
+- Docs/Specification (Specification initially, then Docs as development progresses)
+- Version 1 Docs (https://github.com/theupdateframework/notary/tree/master/docs initially, then /docs/v1/(latest?))
+- Community
+- Getting started
+- Specification (https://github.com/notaryproject/notaryproject initially)
+
+
+### Moving and consolidating existing documentation
+
+- We recommend that the V1 Notary Project that is currently housed in the [The Update Framework](https://github.com/theupdateframework/notary/) GitHub organization be moved into the [Notary Project](https://github.com/notaryproject) GitHub organization.
+- The V1 documentation makes reference to Docker (which was the original home of the documentation). As the V1 docs are moved from their original location, references to things like "the Docker Notary documenation" should be removed.
+- Even if the V1 Notary repo isn't moved, the website MVP should link to V1 notary where it is now, with no changes.
+- Docs+Code combined repositories are a long-term risk. We strongly recommend decoupling these into their own repositories
+   - As the website is being built, we recommend the existing V2 documentation be moved out of the https://github.com/notaryproject/notaryproject repo, and into the new documentation repository: https://github.com/notaryproject/notaryproject.dev
```

Comment:
I agree with the suggestion to use active voice, in this case (and throughout the doc) I was trying to avoid "we" except in reference to CNCF recommendations (and even then I'm not super keen on "we"). I'll see if I can rewrite this line to both use active voice, and avoid "we."